### ISSUE TITLE
Make container rootless

### ISSUE THREAD

**Issue by kereis** [kereis/traefik-certs-dumper#85]

**Comment by kereis** [comment]
I don't know how feasible this one is, as we have functionality to change ownership of files through env. variables. Leveraging this to a non-root user would possibly also require using `sudo`, `gosu` or something else. At this point I don't really know if it makes sense to change the container user if root is required for volume folders and changing permissions.

**Comment by kereis** [comment]
See branch https://github.com/kereis/traefik-certs-dumper/tree/feature/%2385_rootless

**Comment by benbender** [comment]
Yes, and its even more secure as it happens even before the container is started. And it enables the optionto drop the capabilities to do chgrp/chown for the container all together. As I said, most of that stuff should be the responsibility of the runtime…
